processor/src/main/java/org/mapstruct/ap/internal/model/common/Type.java

```diff
@@ -189,6 +189,11 @@ public boolean isVoid() {
         return isVoid;
     }
 
+    public boolean isAbstract() {
```

Comment:
Yes it can kick in for interfaces also. I will update the 2 places where we use this to have one message for interface and abstract class